libc-test/build.rs

```diff
@@ -387,6 +400,15 @@ fn main() {
             "KERN_KDENABLE_BG_TRACE" if apple => true,
             "KERN_KDDISABLE_BG_TRACE" if apple => true,
 
+            // uClibc doesn't support a LOT of constants, because its latest release
+            // was before they coud be implemented yet
+            "LC_PAPER" | "LC_NAME" | "LC_ADDRESS" | "LC_TELEPHONE" | "LC_MEASUREMENT" | "LC_IDENTIFICATION" |
+            "LC_PAPER_MASK" | "LC_NAME_MASK" | "LC_ADDRESS_MASK" | "LC_TELEPHONE_MASK" | "LC_MEASUREMENT_MASK" | "LC_IDENTIFICATION_MASK" | "LC_ALL_MASK" |
+            // not entirely sure why these don't work...
+            "LC_CTYPE_MASK" | "LC_NUMERIC_MASK" | "LC_TIME_MASK" | "LC_COLLATE_MASK" | "LC_MONETARY_MASK" | "LC_MESSAGES_MASK" |
+            "MADV_MERGEABLE" | "MADV_UNMERGEABLE" | "MADV_HWPOISON" | "IPV6_ADD_MEMBERSHIP" | "IPV6_DROP_MEMBERSHIP" | "IPV6_MULTICAST_LOOP" | "IPV6_V6ONLY" |
+            "MAP_STACK" | "RTLD_DEEPBIND" | "MAP_HUGETLB" |
+            "SOL_IPV6" | "SOL_ICMPV6" if uclibc => true,
```

Comment:
I'm pretty sure these need to be ignored, because you can grep them in the codebase, but compiling fails, so I'm assuming these are optional features that my test case (OpenWRT) doesn't have built in. At least to me it seems like ignoring them is the best solution for now